src/nb_enum.cpp

```diff
@@ -150,6 +154,28 @@ bool enum_from_python(const std::type_info *tp, PyObject *o, int64_t *out, uint8
     if (!t)
         return false;
 
+    if ((t->flags & (uint32_t) type_flags::is_flag_enum) !=0) {
```

Comment:
Please follow the code style of the rest of this file. Spaces on both sides of binary operators, space after `if` (several instances of that further below), etc.

Reply:
+1